build/root/Makefile.generated_files

```diff
@@ -501,23 +501,26 @@ API_RULE_CHECK_FAILURE_MESSAGE = "ERROR: \n\t $(1) API rule check failed. Report
 OPENAPI_GEN := $(BIN_DIR)/openapi-gen
 
 KUBE_KNOWN_VIOLATION_FILENAME := api/api-rules/violation_exceptions.list
+AGGREGATOR_KNOWN_VIOLATION_FILENAME := api/api-rules/aggregator_violation_exceptions.list
 APIEXTENSIONS_KNOWN_VIOLATION_FILENAME := api/api-rules/apiextensions_violation_exceptions.list
 CODEGEN_KNOWN_VIOLATION_FILENAME := api/api-rules/codegen_violation_exceptions.list
 SAMPLEAPISERVER_KNOWN_VIOLATION_FILENAME := api/api-rules/sample_apiserver_violation_exceptions.list
 
 APIMACHINERY_DEFAULT_TAG_FILES := vendor/k8s.io/apimachinery/pkg/apis/meta/v1/% vendor/k8s.io/apimachinery/pkg/runtime/% vendor/k8s.io/apimachinery/pkg/version/%
 
 KUBE_OPENAPI_TAG_FILES :=            $(filter-out vendor/k8s.io/code-generator/% vendor/k8s.io/sample-apiserver/%, $(ALL_K8S_TAG_FILES))
+AGGREGATOR_OPENAPI_TAG_FILES :=      $(filter $(APIMACHINERY_DEFAULT_TAG_FILES) vendor/k8s.io/kube-aggregator/%, $(ALL_K8S_TAG_FILES))
```

Comment:
openapi schemas for aggregator is for now generated/embeded from the kube-apiserver https://github.com/kubernetes/kubernetes/pull/50864#discussion_r133905283. moving aggregator schemas to a separate package is probably a positive change.

Reply:
Separated OPENAPI change to a new PR https://github.com/kubernetes/kubernetes/pull/91537